src/lib/static/recidivismSlider.js

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
I don't totally follow yet why this is under `lib/static/` and not under `src/components/Recidiviz/RecidivizSlider/`?

Reply:
💯 

My thinking here is that these should _probably_ be refactored into our own code and components in a subsequent PR. If we do that, we'll take ownership for the code, add tests, properly lint, etc. Until then, I was treating this like purely third-party code which it is, for the most part.